Makefile

```diff
@@ -0,0 +1,112 @@
+DOCKER_BUILD_CMD=docker build
+PROTOCOL_VERSION=2.0
+DOCKERFILES=docker
+CONTEXT=.
+# Additional packages versions
+DOCKER_VERSION_MIN=27.1.2
+NODE_VERSION_MIN=20.17.0
+YARN_VERSION_MIN=1.22.19
+RUST_VERSION=nightly-2024-08-01
+SQLX_CLI_VERSION=0.8.1
+FORGE_MIN_VERSION=0.2.0
+
+# Versions and packages checks
+check-nodejs:
+	@command -v node >/dev/null 2>&1 || { echo >&2 "Node.js is not installed. Please install Node.js v$(NODE_VERSION_MIN) or higher."; exit 1; }
+	@NODE_VERSION=$$(node --version | sed 's/v//'); \
+	if [ "$$(echo $$NODE_VERSION $(NODE_VERSION_MIN) | tr " " "\n" | sort -V | head -n1)" != "$(NODE_VERSION_MIN)" ]; then \
+		echo "Node.js version $$NODE_VERSION is too low. Please update to v$(NODE_VERSION_MIN)."; exit 1; \
+	fi
+
+check-yarn:
+	@command -v yarn >/dev/null 2>&1 || { echo >&2 "Yarn is not installed. Please install Yarn v$(YARN_VERSION_MIN) or higher."; exit 1; }
+	@YARN_VERSION=$$(yarn --version); \
+	if [ "$$(echo $$YARN_VERSION $(YARN_VERSION_MIN) | tr " " "\n" | sort -V | head -n1)" != "$(YARN_VERSION_MIN)" ]; then \
+		echo "Yarn version $$YARN_VERSION is too low. Please update to v$(YARN_VERSION_MIN)."; exit 1; \
+	fi
+
+check-rust:
+	@command -v rustc >/dev/null 2>&1 || { echo >&2 "Rust is not installed. Please install Rust v$(RUST_VERSION)."; exit 1; }
+	@command -v rustup install $$RUST_VERSION  >/dev/null 2>&1
```

Comment:
I think this line should be without `-v`

Reply:
`rustup` w\o `-v` somehow fails
```
make build-all
make: *** [check-rust] Error 1
```